### ISSUE TITLE
Rich text editor in InlinePanels broken in 1.5

### ISSUE THREAD

**Issue by thearrow** [torchbox/wagtail#2662]
Thanks for all your great work on Wagtail!

Just did a pip install of 1.5 into an existing project, now the rich text fields display as plain text (```<p>``` tags visible) and the editor buttons do not appear:

<img width="353" alt="screen shot 2016-06-01 at 10 00 53 am" src="https://cloud.githubusercontent.com/assets/447736/15716519/d8914642-27df-11e6-94fa-0f8c9ab7b550.png">

Console errors say ```Uncaught ReferenceError: makeHalloRichTextEditable is not defined``` once for each rich text field:

<img width="307" alt="screen shot 2016-06-01 at 10 01 01 am" src="https://cloud.githubusercontent.com/assets/447736/15716541/f3dfa3b2-27df-11e6-9fe9-a87cb0556121.png">

I ran ```python manage.py collecstatic --noinput --clear``` and ```python manage.py migrate``` after the upgrade... did I miss another step or is this broken?

**Also: just confirmed that downgrading to 1.4.5 fixes this issue for me - rich text editor works properly.**

**Comment by gasman** [comment]
Hi @thearrow - the `makeHalloRichTextEditable` function is defined in the file `hallo-bootstrap.js`, which is new in Wagtail 1.5. If you reload the page while you have the developer tools open at the 'Network' tab, do you see `hallo-bootstrap.js` listed in the downloaded files (and if so, does it show up as status '200')?

**Comment by thearrow** [comment]
Hey @gasman! 

Unfortunately I don't see ```hallo-bootstrap.js``` anywhere in the network tab (running ```runserver``` with 1.5):

<img width="409" alt="screen shot 2016-06-01 at 10 46 38 am" src="https://cloud.githubusercontent.com/assets/447736/15717916/4bc1d1da-27e6-11e6-96ab-526d1e674ccc.png">

**Comment by gasman** [comment]
Just to clarify - this is on the main Page editing view, rather than one of the more peripheral bits of Wagtail (such as a site settings view set up through `wagtail.contrib.settings`)...?

Is your 'body' field a direct field on your page model, or is it inside a StreamField or an InlinePanel?

**Comment by thearrow** [comment]
This is on the main Page editing view, yes.

The RichTextField **is inside an InlinePanel**, suppose I should have mentioned that in the original issue.

Abbreviated code:
```python
class HomePage(Page):
    ...
    content_panels = Page.content_panels + [
        ...
        InlinePanel('info_items', label="Info Items"),
        ...


class InfoItem(Orderable):
    page = ParentalKey(HomePage, related_name='info_items')
    body = RichTextField(blank=True)
    ...
    content_panels = Page.content_panels + [
        FieldPanel('body'),
        ...
    ]
```

**Comment by m1kola** [comment]
Fixed in #2673
